catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -191,6 +202,19 @@ protected Object executeWithSubject() throws Exception {
             }
         }
 
+        ForkJoinPool forkJoinPool = new ForkJoinPool(multithreaded);
+
+        Stream<Path> ingestStream = forkJoinPool.submit(() -> Files.walk(inputFile.toPath(),
```

Comment:
I don't think this forkJoinPool actually does anything. I think you can replace it with just ```Stream<Path> ingestStream = Files.walk(inputFile.toPath(),FileVisitOption.FOLLOW_LINKS);```